src/views/Hero/Hero.scss

```diff
@@ -1,11 +1,57 @@
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
+
 .HeroView {
-  height: 100%;
+  display: flex;
+  overflow: hidden;
+  justify-content: center;
+  min-height: 100%;
 
   &__homeView {
     background: linear-gradient(180deg, var(--wedgewood__gradient) 0%, var(--waterleaf) 100%);
-    height: 90%;
+    min-height: 90%;
     border-bottom-left-radius: 50px;
     border-bottom-right-radius: 50px;
     box-shadow: 0 10px 16px rgba(0, 0, 0, 0.25);
   }
+
+  &__container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    width: 100%;
+    max-width: 1200px;
+    margin: 5em 3em;
+
+    @include responsiveness.respond(phone) {
+      margin: 1em;
+    }
+  }
+
+  &__visual {
+    // postcard styling
+    max-width: 500px;
+    margin: 0 auto;
+  }
```

Comment:
Is this class also necessary when the `PostcardStack` comprises its own component?

Reply:
This might be necessary because remember that `PostcardStack` will only style the component itself independent of where it goes on the hero page.

Reply:
Ahh, that makes sense - we'll have to re-ensure that the `PostcardStack` composes only of itself rather than the container surounding it.